siliconcompiler/design.py

```diff
@@ -624,6 +668,55 @@ def active_fileset(self, fileset: str):
         with self.active(fileset=fileset):
             yield
 
+    def get_fileset_mapping(self,
```

Comment:
How about shortening the method name to be get_fileset(), this would be consistent  with read_fileset and write_fileset in that they resolve the whole tree.